listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx

```diff
@@ -14,6 +14,7 @@ export type UserTopEntityProps = {
   entity: Entity;
   user?: ListenBrainzUser;
   apiUrl: string;
+  newTerminology: string;
```

Comment:
just `terminology` as the variable name should be fine.

Reply:
Okk fine just updating in few mins